applications/plugins/Geomagic/CMakeLists.txt

```diff
@@ -9,7 +9,9 @@ find_package(SofaLoader REQUIRED)
 find_package(OpenHaptics REQUIRED)
 
 if( NOT OPENHAPTICS_FOUND )
-    message( SEND_ERROR "OpenHaptics library not found, GeomagicDriver cannot compile")
+    message( WARNING "OpenHaptics library not found, GeomagicDriver cannot compile")
+elseif( NOT OPENHAPTICS_FOUND )
+    add_definitions("-DHAS_OPENHAPTICS")
```

Comment:
See previous comment.

Reply:
so sofa_find_package((OpenHaptics QUIET) in Geomagic CMakeLists
will create GEOMAGIC_HAVE_OPENHAPTICS ?
how can I check the spelling  of: (${MODULENAME}HAVE${PACKAGENAME})

Reply:
_"so sofa_find_package((OpenHaptics QUIET) in Geomagic CMakeLists
will create GEOMAGIC_HAVE_OPENHAPTICS ?"_
Yes

_"how can I check the spelling of: (${MODULENAME}HAVE${PACKAGENAME})"_
I guess in the SofaMacro in Sofa.Config but in your case it should be GEOMAGIC_HAVE_OPENHAPTICS